<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>710</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>710</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>661</height>
    </rect>
   </property>
   <widget class="QWidget" name="mainBody" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>721</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f1f5f9;</string>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>410</y>
       <width>721</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: '#ffffff';</string>
     </property>
     <widget class="QTextEdit" name="result_text">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>641</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 4px;
border-color: #b0c2de;
border-radius: 4px;</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="cam">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 2px;
border-radius: 7px;
border-color: #768085;
background-color: #768085;
</string>
     </property>
     <property name="text">
      <string>CAM 영역</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="headBody" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;</string>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="home_title">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>120</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 15pt &quot;BM Dohyeon&quot;;</string>
      </property>
      <property name="text">
       <string>Sign Language</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="home_btn">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>10</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: #d2def1;
	border-radius: 4px
}

QPushButton:hover{
	background-color: #C0D2F2;
	border-radius: 4px
}</string>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/home.png</normaloff>img/home.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>13</width>
       <height>13</height>
      </size>
     </property>
    </widget>
    <widget class="QCheckBox" name="onoff_btn">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>10</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #d2def1;
border-radius: 4px</string>
     </property>
     <property name="text">
      <string>ON / OFF</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
